src/xcode/ENA/ENA/Source/Client/HTTP Client/CachingHTTPClient.swift

```diff
@@ -12,7 +12,7 @@ class CachingHTTPClient: AppConfigurationFetching, StatisticsFetching, LocalStat
 		case dataVerificationError(message: String?)
 	}
 
-	typealias AdministrativeUnit = String
+	typealias groupID = String
```

Comment:
Types should be capitalized! To make it perfect I would add a comment what a 'group id' is like:
> Group ID cluster one to n federal states as regional groups.

(Or something more appropriate)

Reply:
Maybe you call this `GroupIdentifier` to not be confused with the existing `groupID` variables.